### ISSUE TITLE
Invalid DCLP EpiDoc files: attribute "type" not allowed here (7161 cases / biblScope)

### ISSUE THREAD

**Issue by Edelweiss** [DCLP/dclpxsltbox#182]
* [555](https://github.com/DCLP/idp.data/blob/dclp/DCLP/1/555.xml#L87) (expected attribute "ana", "cert", "change", "copyOf", "corresp", "exclude", "facs", "from", "n", "next", "prev", "rend", "rendition", "resp", "sameAs", "select", "style", "synch", "to", "unit", "xml:base", "xml:id", "xml:lang" or "xml:space")
* [99548](https://github.com/DCLP/idp.data/blob/dclp/DCLP/100/99548.xml#L85) (expected attribute "ana", "cert", "change", "copyOf", "corresp", "exclude", "facs", "from", "n", "next", "prev", "rend", "rendition", "resp", "sameAs", "select", "style", "synch", "to", "unit", "xml:base", "xml:id", "xml:lang" or "xml:space")
* [99576](https://github.com/DCLP/idp.data/blob/dclp/DCLP/100/99576.xml#L83) (expected attribute "ana", "cert", "change", "copyOf", "corresp", "exclude", "facs", "from", "n", "next", "prev", "rend", "rendition", "resp", "sameAs", "select", "style", "synch", "to", "unit", "xml:base", "xml:id", "xml:lang" or "xml:space")

**Comment by Edelweiss** [comment]
[example file old](https://github.com/DCLP/idp.data/blob/dclp/DCLP/1/555.xml#L87)

```
/tei:TEI/tei:text/tei:body/tei:div[@type="bibliography"][@subtype="principalEdition"]/tei:listBibl/tei:bibl[@type="publication"][@subtype="principal"]/tei:biblScope/@type
```

[example file new](https://github.com/DCLP/idp.data/blob/hd/DCLP/1/555.xml#L87)

```
/tei:TEI/tei:text/tei:body/tei:div[@type="bibliography"][@subtype="principalEdition"]/tei:listBibl/tei:bibl[@type="publication"][@subtype="principal"]/tei:biblScope/@unit
```

**Comment by paregorios** [comment]
The three examples above come through the current transform just fine into the HTML because the transform for principal edition just takes the text content of the subtree at the ```<bibl>``` level, ignoring any subordinate tagging (e.g., ```<biblScope>```), but including the subordinate text nodes. 

There are other portions of the XSLT, however, that do pay attention to the ```<biblScope>``` tag and associated attributes. I'm going to try to find some diagnostic cases (outside principalEdition) and add them to this ticket so we can assess impact and mitigate. 

##Other examples:

 - 59436 (ancient work): note that that one also has illustrations: bibl type=printed and bibl type=online (it's ignoring the latter)
 - 63748 (in commentary; I've merged in @leoba's code changes from the #156 branch so we can see what it's doing with those)

I don't see any other cases, but I do see shared code for bibliography that is using biblscope across multiple collections.

**Comment by paregorios** [comment]
Over to @jcowey @rla2118 and @HolgerEssler for review on the basis of items posted at http://dclp.github.io/dclpxsltbox/output/dclp/, including:

- [555](http://dclp.github.io/dclpxsltbox/output/dclp/1/555.html) (NB principal edition)
- [59436](http://dclp.github.io/dclpxsltbox/output/dclp/60/59436.html) (NB ancient work)
- [63748](http://dclp.github.io/dclpxsltbox/output/dclp/64/63748.html) (NB introduction, passim)
- [99548](http://dclp.github.io/dclpxsltbox/output/dclp/100/99548.html) (NB principal edition)
- [99576](http://dclp.github.io/dclpxsltbox/output/dclp/100/99576.html) (NB principal edition)
- [69008](http://dclp.github.io/dclpxsltbox/output/dclp/70/69008.html) (NB in note 7: B. Kramer, APF 41 (1995)220-1, which renders exactly what's encoded; encoding needs changing if the lack of a comma and space is unacceptable).
